src/index.ts

```diff
@@ -118,6 +123,16 @@ export default class VideoReporter extends WdioReporter {
         if (this.#usingAllure) {
             process.on('exit', () => this.onExit.call(this))
         }
+
+        const specName = path.basename(runner.specs[0], path.extname(runner.specs[0])) 
+      
+        const browserName = typeof runner.capabilities.browserName === 'string' ? runner.capabilities.browserName : 'unknown-browser';
```

Comment:
We have some logic in the `@wdio/spec-reporter` that reliably gets you a browser name or the mobile app. I think it might make sense to move some of these utilities into the `@wdio/reporter` package and allow other reporters like this one to use it. Mind raising an issue for it in the main repo?

Reply:
sure, https://github.com/webdriverio/webdriverio/issues/14446